modules/global/src/com/haulmont/cuba/core/global/EmailInfo.java

```diff
@@ -38,8 +38,15 @@
 
     /**
      * Recipient email addresses separated with "," or ";" symbol.
+     * <p>
+     * Flag {@code sendInOneMessage} is for backward compatibility with previous CUBA versions.
+     * If {@code sendInOneMessage = true} then one message will be sent for all recipients and it will include CC and BCC.
+     * Otherwise CC and BCC are ignored and multiple messages by the number of recipients in addresses will be sent.
```

Comment:
Hi - I am the user who submitted this request last year and I am happy to see progress. I am concerned about the splitting of messages. Isn't the typical behavior of an email like this: suppose we send 1 email with 2 addresses in the "to" field and 2 more addresses in the "cc" field. A single email is sent to all 4 people and any one of those will see all 4 addresses in the message header and can choose "reply all" and send reply back to the other three recipients?? I will test the current version to see how it behaves with multiple addresses in the "to" field.

Reply:
I just confirmed in 6.10.x a message with 2 addresses in the "to" field will generate 2 separate emails with one address each. From my perspective this is a bug. If separate emails are desired the developer can create and send one per address. I believe the ability to send 1 message with multiple recipients should be the default. If this issue should be submitted as a separate bug, I am OK with that and can post a message in the discussion forum to start the process.

Reply:
Yes, I understand. As long as there is a way to send a single message (like `sendInOneMessage=true`) that is OK for me. I should have read the PR more carefully.
Thanks!